hw/ip/pwrmgr/data/pwrmgr_sec_cm_testplan.hjson

```diff
@@ -25,81 +25,187 @@
+              mixed valid and invalid values.
+
+            **Check**:
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.lc_hw_debug_en_i and tb.dut.lc_dft_en_i
+            - Add assertion to check whether rom_intg_chk_dis
+              is set to '1' only when lc_dft_en_i or lc_hw_debug_en_i
+              is high.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_lc_ctrl_intersig_mubi"]
     }
     {
       name: sec_cm_rom_ctrl_intersig_mubi
-      desc: "Verify the countermeasure(s) ROM_CTRL.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) ROM_CTRL.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Use comprehensive stimulus - reset and wakeup -
+              as background traffic to ensure this counter measure
+              is valid for various states of fast and slow fsm.
+            - Drive rom_ctrl_i with mixed valid and invalid values.
+
+            **Check**:
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.rom_ctrl_i
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_rom_ctrl_intersig_mubi"]
     }
     {
       name: sec_cm_rstmgr_intersig_mubi
-      desc: "Verify the countermeasure(s) RSTMGR.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) RSTMGR.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Drive tb.dut.sw_rst_req_i with mixed valid and invalid values
+
+            **Check**:
+            - See sw rst only happens when dut gets valid value.
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.sw_rst_req_i
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_rstmgr_intersig_mubi"]
     }
     {
       name: sec_cm_esc_rx_clk_bkgn_chk
-      desc: "Verify the countermeasure(s) ESC_RX.CLK.BKGN_CHK."
+      desc: '''Verify the countermeasure(s) ESC_RX.CLK.BKGN_CHK.
+
+            **Stimulus**:
+            - At FastPwrStateActive state, create escalation clock
+              or reset failure by stopping clock or asserting reset.
+
+            **Check**:
+            - Expecting fatal alert event. After alert agent process
+              the alert by asserting escalation reset, see if dut
+              is back to normal operation state.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_esc_clk_rst_malfunc"]
     }
     {
       name: sec_cm_esc_rx_clk_local_esc
-      desc: "Verify the countermeasure(s) ESC_RX.CLK.LOCAL_ESC."
+      desc: '''Verify the countermeasure(s) ESC_RX.CLK.LOCAL_ESC.
+
+            This is triggered by common cm primitives (SecCmPrimCount).
+            (https://github.com/lowRISC/opentitan/blob/master
+            /hw/dv/sv/cip_lib/doc/index.md#security-verification
+            -for-common-countermeasure-primitives)
+
+            **Check**:
+            - Detect fast state transition to FastPwrStateResetPrep.
```

Comment:
hmm this does not sound quite right. If it's `esc_rx`, i think this is really the same test as the one above.

Reply:
It is indeed. I thought it is about  u_esc_timeout.esc_timeout
from description in RTL.
https://github.com/lowRISC/opentitan/blob/4048a11acbddd921532f3d205920ddead7d5fb1b/hw/ip/pwrmgr/rtl/pwrmgr.sv#L122
https://github.com/lowRISC/opentitan/blob/master/hw/ip/pwrmgr/rtl/pwrmgr.sv#:~:text=rstreqs%5BResetEscIdx%5D%20%3D%20esc_rst_req%20%7C-,esc_timeout,-%3B